webrtc.html

```diff
@@ -6996,9 +6995,9 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           for which candidate gathering began.</p>
         </li>
         <li>
-          <p>Set <var>transport</var>'s <code><a data-link-for=
-          "RTCIceTransport">gatheringState</a></code> to <code><a
-          data-link-for="RTCIceGathererState">gathering</a></code>.</p>
+          <p>Set <var>transport</var>'s <a>[[\IceTransportGatheringState]]</a>
```

Comment:
nit: Could be called `[[IceGathererState]]`.

Reply:
Agree with Taylor.